docs/runbooks/OutdatedVirtualMachineInstanceWorkloads.md

```diff
@@ -39,47 +40,67 @@ spec:
-
-### Automated workload updates are enabled but VMIs are still out-of-date
-
-Identify the VMIs that are out of date. Check each impacted VMI to see if they are live migratable or not by looking for the ```LiveMigratable``` condition within the VMI's status.
-
-If a VMI is not live migratable, and the `evict` method is not chosen as a `workloadUpdateMethods` value on the KubeVirt CR, then the only path forward is to stop the VMI. If the VMI is controlled by a corresponding VM with `runStrategy: always` set, then a new VMI will immediately spin up in an updated virt-launcher pod to replace the stopped VMI. This is the equivalent of a restart action.
-
-If the VMI is live migratable and the migration is failing, then the previous option of stopping the VMI is still possible here, but it will result in the workload being interrupted.
-
-Below is an example of how to stop a VM named `my-vm` in namespace `my-namespace` using `virtctl`
-
-```bash
-virctl stop --namespace my-namespace my-vm
+3. Check each outdated VMI to determine whether it is live-migratable:
+  ```bash
+  $ kubectl get vmi <vmi> -o yaml
+  ```
+<!--putting in line breaks so that the yaml is not mangled in conversion. This will have to be fixed post-conversion.-->
+Example output:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+...
+  status:
+    conditions:
+    - lastProbeTime: null
+      lastTransitionTime: null
+      message: cannot migrate VMI which does not use masquerade to connect to the pod network
+      reason: InterfaceNotLiveMigratable
+      status: "False"
+      type: LiveMigratable
 ```
 
-### Manually updating VMIs
-
-To manually update VMIs, either manually create migration objects for migratable VMIs (non distructive), or manually stop VMIs (distructive) which are not live migratable.
+## Mitigation
 
-With live migration, the VMIs will be migrated into an updated virt-launcher container.
+### Configure automated workload updates
 
-With stop (and potentially restart when the VMI is controlled by a VM), any replacement VMIs will be started in updated virt-launcher containers. 
+You can configure automated workload updates in the `KubeVirt` CR. The control plane updates the workloads by using one of the following methods:
+- `LiveMigrate`: The VMI is live-migrated into a new pod with updated components.
+- `Evict`: The VMI pod is shut down. If `runStrategy: always` is configured in the `VirtualMachine` object that controls the VMI, a new VMI spins up in a new pod with updated components.
 
-Below is an example of how to manually execute a live migration by posting a VirtualMachineInstanceMigration object to the cluster that targets a specific running VM. In this example, the VM is called `my-vm` which is running in the namespace `my-namespace`.
+<!--USstart-->
+See [Updating KubeVirt Workloads](https://kubevirt.io/user-guide/operations/updating_and_deletion/#updating-kubevirt-workloads) for more information.
```

Comment:
@enp0s3 @apinnick  Is there a d/s guide for this? If not, perhaps we should consider adding one or a kbase that we can refer to?

Reply:
Hi, I've found [this](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/virtualization/index#updating-openshift-virtualization) in the d/s docmentation.

Reply:
> Hi, I've found [this](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/virtualization/index#updating-openshift-virtualization) in the d/s docmentation.

@enp0s3  What is the "this" you are referring to? Sure, there is D/S documentation. The problem is that our tooling does not allow us to put a link to OpenShift docs within OpenShift modules, only assemblies. It's a real pain. We hope that someday the problem is solved. However, currently, I can only link to a Kbase article in this file. I can't link to anything in DS OpenShift docs. :-(

Reply:
@apinnick Can you create a story to create a small kbase and it can point to the d/s link that @enp0s3 suggested? Do you think its important enough to help with d/s mitigation?